python/lambda-ddb-mysql-etl-pipeline/lambda/dbwrite.py

```diff
@@ -156,40 +156,40 @@ def dynamic_mysql_crud_ops(ast_list, t_suffix, mysql_attrs):
         }
     else:
         # Setting header to None to trick Pandas as it ignores them otherwise
-        df_attrs = pandas.read_csv(ast_list.read()), sep='|', header=None, index_col=False, dtype=str, keep_default_na=False, nrows=1)
+        df_attrs = pandas.read_csv(ast_list.read(), sep='|', header=None, index_col=False, dtype=str, keep_default_na=False, nrows=1)
         # Creates final tuple of attributes from dataframe attributes
         final_attrs = [tuple(x) for x in df_attrs.values]
         final_attrs = final_attrs[0]
         # Extract rows from file and create tuples
-        df_tuples = pandas.read_csv(ast_list.read()), sep='|', na_values=None, keep_default_na=False, dtype=str)
+        df_tuples = pandas.read_csv(ast_list.read(), sep='|', na_values=None, keep_default_na=False, dtype=str)
         df_tuples = df_tuples.apply(tuple, axis=1)
-        
+
         # Compare dwr attrs against s3 attrs to determine ALTER statement requirement and run, if necessary
         for attr in final_attrs:
             if attr in mysql_attrs:
-            logger.info('No new attribute to record')
+                logger.info('No new attribute to record')
             else:
                 logger.info(attr)
                 alter_attr_statement = f"""
                     ALTER TABLE {SCHEMA}.table_{t_suffix} ADD `{attr}` VARCHAR(50)"""
                 cursor_obj.execute(alter_attr_statement)
-        
+
         # Strip empty quotes in df_tuples, transform to 'None' and build final tuples array for INSERT
         tuples_nullified = []
         for tup in df_tuples:
             tuples_nullified.append((tuple(None if elem == '' else elem for elem in tup)))
-        
+
         # Build dynamic attribute strings for INSERT
         attr_str_insert = ""
         attr_var_insert = ""
         for attr in final_attrs:
             if attr != final_attrs[len(final_attrs) -1]:
-        attr_str_insert += "`"+attr+"`, "
-        attr_var_insert += "%s,"
+                attr_str_insert += "`"+attr+"`, "
+            attr_var_insert += "%s,"
```

Comment:
Uncomfortable with this. 

There's a lot of logic change here that I'd like to see tested.

Reply:
The code didn't make sense from an indentation point of view here, so I believe it was bugged to begin with. 

I don't believe it should be in the scope of this PR to resolve a bug like this, so I will revert the changes

Reply:
This is the edit I have the least confidence in.